packages/block-library/src/buttons/edit.native.js

```diff
@@ -106,9 +105,15 @@ export default function ButtonsEdit( {
 	);
 
 	function onChangeContentJustification( updatedValue ) {
-		setAttributes( {
-			contentJustification: updatedValue,
-		} );
+		return () => {
+			const justification =
```

Comment:
I'm wondering why this has been changed from the original.
Also on the `web` change here: https://github.com/WordPress/gutenberg/pull/28768/files#diff-1ec6503d5cd2292d6fd7edfbea747f6a9211a16c4c63bd6d894446603776beacR45
the click handling function was also changed.. 🤔

Reply:
It was updated to match the function/usage in Navigation. The main difference is when an item is already selected, and gets selected again, it will deselect the item.

Reply:
Updated in #28955 I moved this function down to the component, also the implementation of the onClick was wrong in the component which is why there was an extra function wrapping here (and in nav block). It should be all fixed up in the other PR